dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/CopyContentTypeForm.java

```diff
@@ -0,0 +1,111 @@
+package com.dotcms.rest.api.v1.contenttype;
+
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+
+/**
+ * This form encapsulates the info for contentlet copy
+ * @author jsanca
+ */
+@JsonDeserialize(builder = CopyContentTypeForm.Builder.class)
+public class CopyContentTypeForm extends Validated {
+
+    @NotNull
```

Comment:
remove @NotNull

Reply:
This is ok, is the only required property